templates/include/glean/glean.1.schema.json

```diff
@@ -35,7 +35,6 @@
         "app_build",
         "app_display_version",
         "architecture",
-        "client_id",
```

Comment:
I wonder if we could make this optional based on the ping type. We definitely want this to be in our owned pings. This could still be optional in the Parquet schema.

I have no clue if this makes sense at all :)

Reply:
It could be done with an `if`/`then` block in JSON schema 7.  I don't think we're using that draft in the pipeline though.

Reply:
This sounds like a nice feature to include in the `glean_parser`, but maybe not worth it here.